routes/api/follows.js

```diff
@@ -1,25 +1,21 @@
 const express = require('express');
 const router = express.Router();
 const createError = require('http-errors');
-const { Follow } = require("../../db/models");
+const { sequelize, Follow } = require("../../db/models");
 const {asyncHandler} = require('../utils');
 
 
 
 
 
 router.delete('/follows/:id', asyncHandler(async (req, res) => {
-    
+    const follow = await Follow.findOne({where: {id: req.params.id}});
```

Comment:
This can be simplified to
```js
const follow = await Follow.findByPk(req.params.id);
```

Also, include a conditional to check if `follow` is undefined or not. If it is, we should send a 404 error.

Reply:
Line 12 simplified to "const follow = await Follow.findPk(req.params.id);".
Added the following conditional:
if (typeof follow === "undefined" ) {res.status(404)};